doc/book/examples.md

```diff
@@ -98,7 +98,7 @@ class AssertUserIdMatches implements AssertionInterface
         $this->article = $article;
     }
 
-    public function assert(Rbac $rbac)
+    public function assert(Rbac $rbac, $permission = null, $role = null)
```

Comment:
Can we add some typehints? If these types are mixed could we please add PHPDocs about allowed types?